exec/java-exec/src/main/java/org/apache/drill/exec/record/RecordBatchSizer.java

```diff
@@ -260,7 +254,7 @@ public static ColumnSize getColumn(ValueVector v, String prefix) {
 
   public static final int MAX_VECTOR_SIZE = ValueVector.MAX_BUFFER_SIZE; // 16 MiB
 
-  private List<ColumnSize> columnSizes = new ArrayList<>();
+  private Map<String, ColumnSize> columnSizes = CaseInsensitiveMap.newHashMap();
```

Comment:
Tim (@ilooner) made the same change for DRILL-6032 ( #1101 )

Reply:
I'll rebase after this change goes in.